R/explorer.R

```diff
@@ -26,45 +49,58 @@ explorer <- function(dataList=c(), addEnv=T, demo=F) {
     )
   )
 
-  formatFileList<-function(fileList){
-    fileList_formatted = list()
-    i<-1
-    for(f in fileList){
-      df<-get(f)
-      if(class(df)=="data.frame"){
-        fileObj = list(
-          File=f,
-          Rows = dim(df)[1],
-          Columns = dim(df)[2],
+  formatFileList <- function(fileList){
```

Comment:
OK - I think I get it. lists of files are passed for processing when `addEnv` or `demo` are `true`. Lines 64 through 70 are only used when `dataList` is a vector of character.